public/js/helpers/collectData.js

```diff
@@ -0,0 +1,7 @@
+const collectData = (selector) => {
+  let data = {};
+  selector.forEach(element => {
+    data[element] = document.querySelector(`${element}`).value.trim();
```

Comment:
@Fatmasiam You could use `element` in querySelector directly, instead of putting into a template literal 🤷‍♂️